packages/next/src/server/patch-error-inspect.ts

```diff
@@ -443,11 +444,13 @@ function sourceMapError(
 ): Error {
   // Create a new Error object with the source mapping applied and then use native
   // Node.js formatting on the result.
+  // The vm may reference mangled symbols in error messages, so deobfuscate it.
+  const message = deobfuscateText(error.message)
```

Comment:
What is this doing? The error message should be free of these to begin with. This is hit in prod so it shouldn't have any dev magic.